media/csv/standards.csv

```diff
@@ -1,37 +1,37 @@
-Competency,Active (y/n),Competency_Abbreviation,Lens,Lens_Description,Source,Description_Lede,Competency_Description,Mastery,End1,End2,End3,End4,End5,End6,Dev1,Dev2,Dev3,Dev4,Dev5,Dev6,Time1,Time2,Time3,Time4,Time5,Time6

```

Comment:
I think that the CSV library we are using allows columns to be moved around without consequence.